README.md

```diff
@@ -11,11 +11,7 @@ To use, add following code on your `.pre-commit-config.yaml`
 ```yaml
 repos:
   - repo: https://github.com/alice-biometrics/alice-precommit
-    rev: v1
+    rev: v1.0.0
```

Comment:
This is very important. As the repo doesn't have any tag yet when we are testing the hook in the target repo we need to specify the commit.